Added tooling to generate React wrappers for Omni Components (#108)
capitec/omni-components: .tooling/react-wrapper/README.md

<br>
<p align="center">
  <img src="https://raw.githubusercontent.com/capitec/omni-components/develop/.tooling/readme/logos/omni.png" alt="Logo" width="128" height="128" />
</p>

<h3 align="center">Omni Components React Wrappers</h3>
<p align="center"><strong><code>@capitec/omni-components-react</code></strong></p>
<p align="center">Wrapper React components for custom web components provided by the <strong><code>@capitec/omni-components</code></strong> package</p>

<br />

[](#Overview)

## Overview

> This package is generated on the publish of the `@capitec/omni-components` package and is not authored manually.

Reviewer: ```suggestion
This package is auto-generated on the publish of the `@capitec/omni-components` package.
```
